crypto/tests/test_check.c

```diff
@@ -58,6 +57,7 @@
 #include "monero/monero.h"
 #include "nem.h"
 #include "nist256p1.h"
+#include "options.h"
```

Comment:
Let's keep `options.h` at the top. I think it contains parameters that might affect other includes and in this case the include order does matter.

Reply:
Done. The new position of `#include <check.h>` is OK?

Maybe we want to add `clang-format off` for the includes?

Reply:
`check.h` should be with no side effects, so the change is fine.